### PR TITLE
Add a basic client for sending emails via gmail SMTP

### PR DESCRIPTION
**Pull request by axmb**
Basic idea is that we'll register addressees per-source, and send emails on relevant updates in our sources controller.

Using GMail's public SMTP server, because YAGNI. We can still add custom domains, IIUC, and if someone's feeling particularly passionate later on we can run/use our own SMTP server instead.

Re: #978

### CODE REVIEW

File: verification/curator-service/api/src/clients/email-client.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        return new EmailClient(service, transport);
+    }
+
+    /**
+     * Sends an email via the configured transport.
+     *
+     * @param addressees - Addressees to whom the email is sent.
+     * @param subject - Subject of the email.
+     * @param text - Body/content of the email.
```
Comment (attwad) [inline verification/curator-service/api/src/clients/email-client.ts:68]: can this be html? or just raw text?

**Review by attwad — APPROVED** [review]

**Comment by attwad** [comment]
Reference a github issue perhaps?

File: verification/curator-service/api/src/clients/email-client.ts (hunk 1 above)
Comment (axmb) [reply verification/curator-service/api/src/clients/email-client.ts:68]: Text has to be plaintext. There's an `html` option, too, which I'm hoping to quickly poke UX for. :point_right:

**Comment by axmb** [comment]
> Reference a github issue perhaps?

Can do!

**Pull request merged**